packages/o/openvpn3/package.yml

```diff
@@ -0,0 +1,46 @@
+name       : openvpn3
+version    : '24.1'
+release    : 1
+source     :
+    - git|https://codeberg.org/OpenVPN/openvpn3-linux.git: v24.1
```

Comment:
Does the tag's source tarball work? https://codeberg.org/OpenVPN/openvpn3-linux/archive/v24.1.tar.gz

Reply:
The project has a submodule ([openvpn3-core](https://github.com/OpenVPN/openvpn3)), with which it is fairly tightly coupled. I could track both as tarballs, but tracking through Git is a bit more convenient.